plexpy/webserve.py

```diff
@@ -1230,24 +1247,25 @@ def get_notification_log(self, **kwargs):
         return json.dumps(notifications)
 
     @cherrypy.expose
+    @cherrypy.tools.json_out()
     @requireAuth(member_of("admin"))
     @addtoapi()
     def clearNotifyLogs(self, **kwargs):
         data_factory = datafactory.DataFactory()
         result = data_factory.delete_notification_log()
+        result = result if result else 'no data received'
 
-        if result:
-            cherrypy.response.headers['Content-type'] = 'application/json'
-            return json.dumps({'message': result})
-        else:
-            cherrypy.response.headers['Content-type'] = 'application/json'
-            return json.dumps({'message': 'no data received'})
+        return {'message': result}
 
     @cherrypy.expose
     @requireAuth(member_of("admin"))
     def clearLogs(self):
-        plexpy.LOG_LIST = []
-        logger.info(u"Web logs cleared")
+        try:
+            open(os.path.join(plexpy.CONFIG.LOG_DIR, 'plexpy.log'), 'w').close()
```

Comment:
I guess if the user clears their logs, and they need help, then it's their fault.

It's not like everyone posts their logs when they have problems anyways...So really no difference =P